spec.html

```diff
@@ -252,15 +252,15 @@ <h1>AsyncIterator.from ( _O_ )</h1>
       </emu-clause>
     </emu-clause>
 
-    <emu-clause id="sec-iteratorprototype-object">
+    <emu-clause id="sec-%iteratorprototype%-object">
```

Comment:
This&nbsp;fragment&nbsp;identifier was&nbsp;wrong, as&nbsp;it’s:&nbsp;<https://tc39.es/ecma262/#sec-%iteratorprototype%-object>.